src/test/java/net/sf/jabref/importer/fileformat/BibtexImporterTest.java

```diff
@@ -0,0 +1,101 @@
+package net.sf.jabref.importer.fileformat;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class BibtexImporterTest {
+
+    private BibtexImporter importer;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
As far as I understand the [javadoc](http://junit.org/javadoc/latest/org/junit/rules/ExpectedException.html) it just declares the standard expected Exception for all tests in that class (which usually is none). 
I will remove it if you think otherwise.